vaadin-grid-flow-parent/vaadin-grid-flow/src/main/java/com/vaadin/flow/component/grid/AbstractColumn.java

```diff
@@ -342,4 +342,42 @@ protected List<Column<?>> getBottomChildColumns() {
                         .collect(Collectors.toList()));
         return columnChildren;
     }
+
+    /**
+     * Sets a custom part name for the header cell.
+     *
+     * @param headerPartName
+     *            the part name to set
+     */
+    public void setHeaderPartName(String headerPartName) {
```

Comment:
I am wondering if the setters should return the column itself as other setters do allow chaining configuration of the column.

Reply:
Done. Also moved the methods to ColumnBase